pkgs/development/libraries/libpointmatcher/default.nix

```diff
@@ -0,0 +1,41 @@
+{stdenv, fetchFromGitHub, cmake, eigen, boost, libnabo}:
+
+let inherit (stdenv.lib) getVersion versionAtLeast; in
+
+assert versionAtLeast (getVersion eigen) "3";
```

Comment:
This is quite unusual, we typically just pass the correct version instead of inspecting the version string of dependencies. Please remove this, unless there's a strong reason to do it.